scripts/ingests/utils.py

```diff
@@ -1,10 +1,253 @@
 import numpy as np
+import re
 from astropy.coordinates import SkyCoord
 import astropy.units as u
 from astroquery.simbad import Simbad
+from astropy.table import Table, vstack
 import warnings
 warnings.filterwarnings("ignore", module='astroquery.simbad')
-import re
+from sqlalchemy import or_
+import ads
+import os
+
+
+def search_publication(db, name: str = None, doi: str = None, bibcode: str = None, verbose: bool = False):
+    """
+    Find publications in the database by matching on the publication name,  doi, or bibcode
+
+    Parameters
+    ----------
+    db
+        Variable referencing the database to search
+    name: str
+        Name of publication to search
+    doi: str
+        DOI of publication to search
+    bibcode: str
+        ADS Bibcode of publication to search
+
+    Returns
+    -------
+    True: if only one match
+    Table containing publications matching name if more than one match
```

Comment:
I worry that `search_publication` returning different types of output can cause issues further along the road. I would prefer if it either always returns a table (even if the table is empty) or always returns a boolean True/False, not the mix of the two.

Reply:
I agree! I was hoping you would tell me to do this. :)